packages/deployer/config/config.json.example

```diff
@@ -3,12 +3,12 @@
     {
       "authorizers": [],
       "contracts": {
-        "AirnodeRrp": "0x5FbDB2315678afecb367f032d93F642f64180aa3"
+        "AirnodeRrp": "0x534DfA0d242e7fDCFD096C2B1a3D70F172008778"
       },
-      "id": "31337",
+      "id": "4",
       "providers": {
-        "local": {
```

Comment:
I've changed this, because deploying on AWS and using local API is not possible. Myabe this should not even be here?